.circleci/config.yml

```
@@ -25,3 +25,124 @@ jobs:
+    docker: 
+      - image: node:latest 
+    steps: 
+      - checkout 
+      - attach_workspace:
+          at: ~/build
+      - run:
+          name: update-npm
+          command: |
+           cd ~
+           npm install npm@latest
+      - run: 
+          name: get remark
+          command: |
+           cd ~
+           npm install remark remark-cli
+      - run: 
+          name: get remark styles
+          command: |
+           cd ~
+           npm install remark-cli@5.0.0 remark-lint@6.0.2 remark-preset-lint-recommended@3.0.2 remark-preset-lint-markdown-style-guide@2.1.2
+      - run: # remark the auto generated changes.md
+          name: remark on autogenerated CHANGES.md
+          command: |
+             cd ~/project
+             if git log -1 --pretty=%s | grep Merge* ; then
+             if git log -1 --pretty=%b | grep release ; then
+             echo "New release, do nothing"
+             mkdir ~/project/src/tmp
+             touch ~/project/src/tmp/empty.txt
+             else
+             mkdir ~/project/src/tmp
+             cat ~/build/src/CHANGES.md
+             cp ~/build/src/CHANGES.md ~/project/src/CHANGES.md
+             ~/node_modules/.bin/remark ~/project/src/CHANGES.md -o ~/project/src/tmp/CHANGES.md
+             ~/node_modules/.bin/remark ~/project/src/tmp/CHANGES.md --frail
+             fi
+             else
+             echo "Not a PR merge commit"
+             mkdir ~/project/src/tmp
+             touch ~/project/src/tmp/empty.txt
+             fi
+      - persist_to_workspace:
+             root: ~/project/src
+             paths: tmp
+
+# Push built changelog to repo
+  push:
+    working_directory: ~/build
+    docker:
+      - image: circleci/openjdk:8-jdk
+    steps:
+      - setup_remote_docker:
+           version: 17.11.0-ce
+      - checkout
+      - attach_workspace:
+            at: ~/build
+      - deploy:
+           name: Changelog deployment
+           working_directory: ~/build
+           command: |
+             if git log -1 --pretty=%s | grep Merge* ; then
+             if git log -1 --pretty=%b | grep release ; then
+             echo "New release, do nothing"
+             else
+             mv ~/build/tmp/CHANGES.md ~/build/src/CHANGES.md
+             git config credential.helper 'cache --timeout=120'
+             git config user.email "franklin.feingold@gmail.com"
+             git config user.name "franklin-feingold"
+             git add ~/build/src/CHANGES.md
+             git commit -m "autobuild CHANGES.md"
+             git push https://${CHANGE_TOKEN}@github.com/bids-standard/bids-specification.git master
+             fi
+             else
+             echo "Not a PR merge commit"
+             fi
+
+workflows:
+  version: 2
+  search_build:
+    jobs:
+      - build
+      - github-changelog-generator
+      - remark:
+          requires:
+               - github-changelog-generator
+      - push:
+          requires:
+               - remark
```

Do you want to run this on every commit? I think limiting to `branch: master` will make more sense:

```suggestion
               - remark
           filters:
               branches:
                   only: master
```

This should probably apply to `github-changelog-generator` and `remark` as well.

---

I have applied this change in my recent commit